tensorflow/inference/docker/2.5/py3/sdk1.16.0/Dockerfile.neuron

```diff
@@ -0,0 +1,152 @@
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV SAGEMAKER_TFS_VERSION="${TFS_SHORT_VERSION}"
+ENV PATH="$PATH:/sagemaker"
+ENV LD_LIBRARY_PATH='/usr/local/lib:$LD_LIBRARY_PATH'
+ENV MODEL_BASE_PATH=/models
+# The only required piece is the model name in order to differentiate endpoints
+ENV MODEL_NAME=model
+ENV DEBIAN_FRONTEND=noninteractive
+
+# nginx + njs
+RUN apt-get update \
+ # TODO: Remove systemd upgrade once it is updated in base image
+ && apt-get -y upgrade --only-upgrade systemd \
+ && apt-get -y install --no-install-recommends \
+    curl \
+    gnupg2 \
+    ca-certificates \
+    emacs \
+    git \
+    unzip \
+    wget \
+    vim \
+    libbz2-dev \
+    liblzma-dev \
+    libffi-dev \
+    build-essential \
+    zlib1g-dev \
+    openssl \
+    libssl1.1 \
+    libreadline-gplv2-dev \
+    libncursesw5-dev \
+    libssl-dev \
+    libsqlite3-dev \
+    tk-dev \
+    libgdbm-dev \
+    libcap-dev \
+    libc6-dev \
+ && curl -s http://nginx.org/keys/nginx_signing.key | apt-key add - \
+ && echo 'deb http://nginx.org/packages/ubuntu/ bionic nginx' >> /etc/apt/sources.list \
+ && apt-get update \
+ && apt-get -y install --no-install-recommends \
+    nginx \
+    nginx-module-njs \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+# Install python3.7
+RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz \
+ && tar -xvf Python-$PYTHON_VERSION.tgz \
+ && cd Python-$PYTHON_VERSION \
+ && ./configure && make && make install \
+ && rm -rf ../Python-$PYTHON_VERSION*
+
+RUN echo "deb https://apt.repos.neuron.amazonaws.com bionic main" > /etc/apt/sources.list.d/neuron.list
+RUN wget -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB | apt-key add -
+RUN apt-get update
+
+RUN apt-get install -y \
+    tensorflow-model-server-neuron=2.5.* \
+    aws-neuron-tools \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN ${PIP} --no-cache-dir install --upgrade \
+    pip \
+    setuptools
+
+# cython, falcon, gunicorn, grpc
+RUN ${PIP} install --no-cache-dir \
+    "awscli<2" \
+    boto3 \
+    cython==0.29.21 \
+    falcon==2.0.0 \
+    gunicorn==20.0.4 \
+    gevent==21.1.1 \
+    requests==2.25.1 \
+    grpcio==1.34.1 \
+    protobuf==3.14.0 \
+# using --no-dependencies to avoid installing tensorflow binary
+ && ${PIP} install --no-dependencies --no-cache-dir \
+    tensorflow-serving-api==2.5.1
+
+# pip install statements have been separated out into multiple sequentially executed statements to
+# resolve package dependencies during installation.
+RUN ${PIP} install "tensorflow_serving_api==2.5.1" --no-deps --extra-index-url https://pip.repos.neuron.amazonaws.com \
+ && ${PIP} install neuron-cc --extra-index-url https://pip.repos.neuron.amazonaws.com \
+ && ${PIP} install "tensorflow-neuron==2.5.1.*" --extra-index-url https://pip.repos.neuron.amazonaws.com
+COPY ./sagemaker /sagemaker
```

Comment:
nit: Please add a space between the previous `RUN` statement and the `COPY` statement for readability.